java/compaction-core/src/main/java/sleeper/compaction/task/CompactionTaskFinishedStatus.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright 2022 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sleeper.compaction.task;
+
+import sleeper.compaction.job.status.CompactionJobStatus;
+
+import java.time.Instant;
+import java.util.List;
+import java.util.Objects;
+
+public class CompactionTaskFinishedStatus {
+    private final Instant finishTime;
+    private final Integer totalJobs;
+    private final double totalRuntime;
+    private final double totalReads;
+    private final double totalWrites;
+    private final double recordsReadPerSecond;
+    private final double recordsWrittenPerSecond;
+
+    public Instant getFinishTime() {
+        return finishTime;
+    }
+
+    public Integer getTotalJobs() {
+        return totalJobs;
+    }
+
+    public double getTotalRuntime() {
+        return totalRuntime;
+    }
+
+    public double getTotalReads() {
```

Comment:
Think this would be better called "getTotalRecordsRead" and it should return a long.